Develop

rachavez opened the pull request:
I've made the translations. I left proper nouns like "TacoTruck.Party (website name)" and "DevProgress" alone.  I also left the bangs (!) at the front of each line alone. I'm not familiar with i18n and Javascript, so I was unable to see the changes in my local site.

There could be some discussion about the translation for "taco truck". I say "camioneta de taco". My mom and dad say "taquero". I would say that's the person working inside the truck, but it could also be the taco truck itself. If you want Spanglish, "troca de taco" make sense, but someone is bound to complain.

If you want more changes, let me know.

rachavez committed:
spanish text

Changed text from English to Spanish. Proper nouns left alone.

rachavez committed:
Merge branch 'develop-rachavez' into develop

flibbertigibbet commented:
Thanks for the translations! The exclamation points at the start of each line should be removed; they were just there to indicate something that should be translated but hadn't been yet.

Other than that, :+1:

rachavez committed:
committing again?

VS Code says I need to commit the file. I thought I already did.

rachavez committed:
Merge branch 'develop-rachavez' into develop

# Conflicts:
#	app/js/i18n/es.json

rachavez committed:
fixed merge from develop branch

I thought I merged changes from develop branch back to my branch. It
looks like I was mistaken. try again.

rachavez committed:
Merge branch 'develop-rachavez' into develop

rachavez commented:
There was some confusion on my fork-and-branch when I went to make the updates. (I thought I updated from the develop branch already? I thought I saved this commit already?). I did remove the bangs (!) at the beginning of each string in the end. If there are still issues or the pull request updates have disappeared, please let me know.

flibbertigibbet commented:
Looks good, thanks! :fireworks: By the way, if you change your preferred browser language to one of the Spanish options, the translations should appear.
